apicast/src/executor.lua

```diff
@@ -0,0 +1,59 @@
+local resty_env = require('resty.env')
+
+if resty_env.get('APICAST_MODULE') then
+  return require('module')
+end
+
+local policy_chain = require('policy_chain')
+local policy = require('policy')
+local linked_list = require('linked_list')
+
+local setmetatable = setmetatable
+
+local _M = { }
+
+local mt = { __index = _M }
+
+-- forward all policy methods to the policy chain
+for _,phase in policy:phases() do
+    _M[phase] = function(self, ...)
+        return self.policy_chain[phase](self.policy_chain, self:context(phase), ...)
+    end
+end
+
+function _M.new()
+    local global_chain = policy_chain.build({
+        'policy.load_configuration', 'policy.find_service', 'policy.local_chain'
```

Comment:
How about extracting this table to a constant ?

Reply:
We can. This would also be controlled by some global configuration file.